examples/consume_sdf/basic.py

```
@@ -0,0 +1,8 @@
+import sdf
+import partcad as pc
+
+part = pc.get_part_sdf(
+    "//pub/examples/partcad/produce_part_sdf:box",
+)
+
+show_object(part)
```

Ruff (0.15.21)

[error] Undefined name `show_object`

(F821)